test/performance/long-running-tests/workloads/long-duration-exp-workload.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sleep-20-sec
+  labels:
+    app.kubernetes.io/name: perf-test
+spec:
+  containers:
+  - name: aw-wait-20
+    image: ubuntu:latest
+    imagePullPolicy: IfNotPresent
+    command: [ "/bin/bash", "-c", "--" ]
+    args: [ "sleep 20" ]
```

Comment:
It is strange to use both `command` and `args`. The `command` field is intended to be something that a user can use _instead of_ the image's entrypoint + the pod's `args`. When I test the combination here, I get a pod in CrashLoopBackOff and every iteration produces an empty log. OTOH, the following produces a Pod that runs fine.

```yaml
apiVersion: v1
kind: Pod
metadata:
  name: sleep-21-sec
  labels:
    app.kubernetes.io/name: perf-test
spec:
  containers:
  - name: aw-wait-20
    image: ubuntu:latest
    imagePullPolicy: IfNotPresent
    command: [ "/bin/bash", "-c", "sleep 21" ]
  restartPolicy: Never
```